### DISCUSSION TITLE
[DOC]: dfp is it all modular pipeline now since 23.03?

### DISCUSSION THREAD

**Discussion by nyck33** [nv-morpheus/Morpheus#1223] [Morpheus]
### How would you describe the priority of this documentation request

Medium

### Describe the future/missing documentation

A example here: https://github.com/nv-morpheus/Morpheus/blob/branch-23.11/docs/source/developer_guide/guides/10_modular_pipeline_digital_fingerprinting.md

Then the production here: https://github.com/nv-morpheus/Morpheus/blob/branch-23.11/examples/digital_fingerprinting/production/README.md

But if it's modular since 23.03, then the second example is modular?  So there is no such thing as a staging/stages version of the dfp pipeline in branch 23.11, is that correct?  

I saw in the docker compose file that it really does seem to be mimicking a production deployment with other containers for kafka, mlflow, etc being built and run which is what I wanted to try, ie. maybe even try running the example in the cloud just to get used to that environment but then came by this modular pipeline for dfp example and got confused.  

### Where have you looked?

see the links above and the GTC roundtable conference and DLI lesson video plus https://docs.nvidia.com/morpheus/developer_guide/guides/6_digital_fingerprinting_reference.html

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct
- [X] I have searched the [open documentation issues](https://github.com/nv-morpheus/Morpheus/issues?q=is%3Aopen+is%3Aissue+label%3Adoc) and have found no duplicates for this bug report

**Comment by jarmak-nv** [comment]
Hi @nyck33!
  
  Thanks for submitting this issue - our team has been notified and we'll get back to you as soon as we can!
  In the mean time, feel free to add any relevant information to this issue.

**Comment by efajardo-nv** [comment]
@nyck33 A DFP pipeline can be built with either stages or modules. The links you provided contain examples of both approaches:

Using stages:
https://github.com/nv-morpheus/Morpheus/blob/branch-23.11/examples/digital_fingerprinting/production/README.md#steps-to-run-example-pipeline

Using modules:
https://github.com/nv-morpheus/Morpheus/blob/branch-23.11/docs/source/developer_guide/guides/10_modular_pipeline_digital_fingerprinting.md#morpheus-pipeline

All example DFP pipelines can be run in the same `docker-compose` environment.

**Comment by jarmak-nv** [comment]
Hey @nyck33 Gonna discussion this one also!

In general we approach issues as concrete items that can be actioned on, since this is mostly a question, it should be a discussion. If something we can improve/fix/add comes out of the discussion we will spawn an issue and triage (understand + decide where in the roadmap) per normal.
